shared/file_utils.go

```diff
@@ -0,0 +1,73 @@
+package shared
```

Comment:
Can you put this under `shared/utils/file_utils.go` and add a TODO to consolidate it with `converters`.

<img width="265" alt="Screenshot 2023-02-20 at 9 50 13 AM" src="https://user-images.githubusercontent.com/1892194/220173266-6f865976-86a0-4684-951d-f30e23405ea5.png">